src/mocks/handlers.ts

```diff
@@ -162,4 +163,9 @@ export const handlers = [
   rest.get(PAGE_CONTENT_API + '/about', async (req, res, ctx) => {
     return res(ctx.json(mockPageContent));
   }),
+
+  // Health Check
+  rest.get(HEALTH_CHECK_API, async (req, res, ctx) => {
+    return res(ctx.text('body')); // currently doesn't return anything, no mocks around this one
```

Comment:
This the server side for this endpoint is https://github.com/osu-wams/dx-server/blob/master/src/index.ts#L97-L103 so you can see the shape and contents of what it returns.